.github/workflows/ci-benchmark.yaml

```diff
@@ -49,10 +49,15 @@ jobs:
       AIRFLOW__ASTRO_SDK__DATAFRAME_ALLOW_UNSAFE_STORAGE: True
       FORCE_COLOR: "true"
       ASTRO_PUBLISH_BENCHMARK_DATA: True
+      USE_GKE_GCLOUD_AUTH_PLUGIN: True
```

Comment:
Don't we need to set this env var also in the Dockerfile?

Reply:
look like currently there is some issue while installing it in GitHub action https://github.com/actions/runner-images/issues/6778#issuecomment-1360360603

Reply:
Not sure I understand. I meant we need to add the env var in the Dockerfile, too.

Reply:
this is currently a workaround to install it in GitHub action and it does not require in docker

Reply:
I don't think it is. It is how it is supposed to work.

The following is from an email, I got this morning:

> To run kubectl with the new plugin prior to the release of v1.26
> 
> To have kubectl use the new binary plugin for authentication instead of using the default provider-specific code, use the following steps.
> 
> Set export USE_GKE_GCLOUD_AUTH_PLUGIN=True in ~/.bashrc (or in Environment variables for Windows).
> Run the following command:
> source ~/.bashrc
> For Windows environments, start a new terminal.
> Update gcloud to the latest version.
> gcloud components update
> Run the following command:
> gcloud container clusters get-credentials CLUSTER_NAME
> Replace the CLUSTER_NAME with the name of your cluster. This will force the config for this cluster to be updated to the Client-go Credential Plugin configuration.
> Full and latest instructions are available at the [Kubectl auth changes in GKE v1.26](https://notifications.google.com/g/p/AM0OveryzzeUSXvh-iaTJQA97-SCYO0kQZz_BS1_D1LqZwStEq7BffHgYZ_XBeZxBY0uptzkbje-62xt2AjtYwwKOU9fkjCZw2WAxufF0kbLQC4b2-T-aXC_uDtymP2NwQT4KHMYY_ZJj5NuteqIcwetMa1U04R5jPueF0aMYlGL6uMw79YXSzw4_7i1nAuferA0FC3g_9UO8wp0j327osRdTp4z4A1rTnmV_XbtqfIWugkM9-4c2eogmNLFsBf1ZhQVdqCMHjVINw) blogpost. Installation instructions above, are copied for convenience.

Reply:
You can also see it in the link of the quote above^

Reply:
can't install it with a component update since the runner image comes with google-cloud-sdk  maybe because it install with apt and throws an error
https://github.com/actions/runner-images/blob/ubuntu20/20230109.1/images/linux/Ubuntu2004-Readme.md
```
Beginning update. This process may take several minutes.
ERROR: (gcloud.components.update) 
You cannot perform this action because the Google Cloud CLI component manager 
is disabled for this installation. You can run the following command 
to achieve the same result for this installation: 
```

I tried to install it with apt only but it was not able to find package 
```
Reading package lists...
Building dependency tree...
Reading state information...
E: Unable to locate package google-cloud-cli-gke-gcloud-auth-plugin
```

Reply:
removed `google-cloud-cli-gke-gcloud-auth-plugin` from docker image since currently, we do not use any kubectl command in docker. We use it only in Makefile that we run on github action host

Reply:
Alright. Do we then need changes to the Dockerfile at all?

Reply:
No, but currently there are lots of `RUN` commands I have just grouped them.